[2023-08-13 12:57:39,693] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: submit_pyspark_script_to_emr.execute_pyspark_script_sensor manual__2023-08-13T12:57:26.213289+00:00 [queued]>
[2023-08-13 12:57:39,711] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: submit_pyspark_script_to_emr.execute_pyspark_script_sensor manual__2023-08-13T12:57:26.213289+00:00 [queued]>
[2023-08-13 12:57:39,712] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-13 12:57:39,713] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-13 12:57:39,714] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-13 12:57:39,736] {taskinstance.py:1377} INFO - Executing <Task(EmrStepSensor): execute_pyspark_script_sensor> on 2023-08-13 12:57:26.213289+00:00
[2023-08-13 12:57:39,743] {standard_task_runner.py:52} INFO - Started process 244 to run task
[2023-08-13 12:57:39,750] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'submit_pyspark_script_to_emr', 'execute_pyspark_script_sensor', 'manual__2023-08-13T12:57:26.213289+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/orchestration/submit_spark_job_to_emr.py', '--cfg-path', '/tmp/tmp_8ag499x', '--error-file', '/tmp/tmp21ewgb5w']
[2023-08-13 12:57:39,754] {standard_task_runner.py:80} INFO - Job 78: Subtask execute_pyspark_script_sensor
[2023-08-13 12:57:39,859] {task_command.py:370} INFO - Running <TaskInstance: submit_pyspark_script_to_emr.execute_pyspark_script_sensor manual__2023-08-13T12:57:26.213289+00:00 [running]> on host airflow_scheduler
[2023-08-13 12:57:40,056] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=submit_pyspark_script_to_emr
AIRFLOW_CTX_TASK_ID=execute_pyspark_script_sensor
AIRFLOW_CTX_EXECUTION_DATE=2023-08-13T12:57:26.213289+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-13T12:57:26.213289+00:00
[2023-08-13 12:57:40,073] {base.py:68} INFO - Using connection ID 'aws_default' for task execution.
[2023-08-13 12:57:40,077] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2023-08-13 12:57:40,078] {base_aws.py:100} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2023-08-13 12:57:40,145] {emr.py:317} INFO - Poking step s-0032803ZW9KTHFLQF3Z on cluster j-1Z3BJELOTWETV
[2023-08-13 12:57:40,484] {emr.py:74} INFO - Job flow currently PENDING
[2023-08-13 12:58:40,456] {emr.py:317} INFO - Poking step s-0032803ZW9KTHFLQF3Z on cluster j-1Z3BJELOTWETV
[2023-08-13 12:58:40,887] {emr.py:74} INFO - Job flow currently FAILED
[2023-08-13 12:58:40,908] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/sensors/base.py", line 277, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/amazon/aws/sensors/emr.py", line 84, in poke
    raise AirflowException(final_message)
airflow.exceptions.AirflowException: EMR job failed for reason None with message Caused by: org.apache.hadoop.shaded.com.google.re2j.PatternSyntaxException: error parsing regexp: Unclosed group at pos 55: `{{ ti.xcom_pull(task_ids='fetch_cluster_id_and_paramns'` and log file s3://emr-data-947775527574/emr_logs/j-1Z3BJELOTWETV/steps/s-0032803ZW9KTHFLQF3Z/stderr.gz
[2023-08-13 12:58:40,913] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=submit_pyspark_script_to_emr, task_id=execute_pyspark_script_sensor, execution_date=20230813T125726, start_date=20230813T125739, end_date=20230813T125840
[2023-08-13 12:58:40,931] {standard_task_runner.py:97} ERROR - Failed to execute job 78 for task execute_pyspark_script_sensor (EMR job failed for reason None with message Caused by: org.apache.hadoop.shaded.com.google.re2j.PatternSyntaxException: error parsing regexp: Unclosed group at pos 55: `{{ ti.xcom_pull(task_ids='fetch_cluster_id_and_paramns'` and log file s3://emr-data-947775527574/emr_logs/j-1Z3BJELOTWETV/steps/s-0032803ZW9KTHFLQF3Z/stderr.gz; 244)
[2023-08-13 12:58:40,962] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-08-13 12:58:40,993] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
